src/Trace/Propagator/HttpHeaderPropagator.php

```diff
@@ -76,8 +76,13 @@ public function extract($headers)
     public function inject(SpanContext $context, $container)
     {
         $header = str_replace('_', '-', preg_replace('/^HTTP_/', '', $this->header));
-        $container[$header] = $this->formatter->serialize($context);
-        return $container;
+        $value = $this->formatter->serialize($context);
```

Comment:
Do you still need the `$container` argument?

Reply:
Ah, ok, it's a part of the interface

Reply:
It's used for other propagators (like grpc). In this case, it's unused because we are injecting the header into current request.